### PR TITLE
Fix announcements compose profile image

### LINKED ISSUE #641: Some small issues with announcement
- ~~The playwright test is failing. It doesn't look like a bug but need to rewrite the test a bit to fix~~ fixed in #644 
- Currently it shows "Announcement" header with no content. We want to show "no announcement" when there isn't any

### LINKED ISSUE #642: Add report icon as seen on near.org posts to announcement and comments of it
Currently:
image

We want to add the three dot icon as seen on near.org posts:
image

It should be add for both announcements and the comments of the announcements. The report functionality should be automatically be there because these announcements/comments are just standard near.org posts/comments, and can be report using near.org's reporting feature

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
Preview [link](https://near.org/megha19.near/widget/app?page=community&handle=devhub-test&tab=announcements) 
Resolves #641
Resolves #642

### CODE REVIEW

File: src/devhub/entity/community/Compose.jsx [hunk 1]
```diff
@@ -39,7 +40,7 @@ function extractMentions(text) {
 
 function extractTagNotifications(text, item) {
   return extractMentions(text || "")
-    .filter((accountId) => accountId !== context.accountId)
+    .filter((accountId) => accountId !== accountId)
```
Comment (ailisp) [inline src/devhub/entity/community/Compose.jsx:43]: this will not work

**Comment by ailisp** [comment]
The preview isn't right, can you check your preview setup?

production:
image


preview:
image

File: src/devhub/entity/community/Compose.jsx (hunk 1 above)
Comment (Megha-Dev-19) [reply src/devhub/entity/community/Compose.jsx:43]: oh yes, good catch, I will update it

**Comment by ailisp** [comment]
Great job on adding report/hide! 
This part of #641 seems not fixed:
> Currently it shows "Announcement" header with no content. We want to show "no announcement" when there isn't any

For example this community has no announcment: https://near.org/megha19.near/widget/app?page=community&handle=webassemblymusic

Could you please address it?

**Comment by Megha-Dev-19** [comment]
@ailisp I don't see any header, or do you want to show the header and a text that no announcement exists?
image

**Comment by ailisp** [comment]
@Megha-Dev-19 Yes, want to show the header and a text that no announcement exists

**Comment by Megha-Dev-19** [comment]
oh now I get it, I misread the requirement, I will update it.

**Comment by Megha-Dev-19** [comment]
image
hey @ailisp, sorry for the delay, I completely forgot about this PR

**Pull request merged**